core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/SampleSingleCellImpl.java

```diff
@@ -0,0 +1,41 @@
+package uk.ac.bbsrc.tgac.miso.core.data.impl;
+
+import java.math.BigDecimal;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import uk.ac.bbsrc.tgac.miso.core.data.SampleSingleCell;
+
+@Entity
+@Table(name = "SampleSingleCell")
+public class SampleSingleCellImpl extends SampleTissueProcessingImpl implements SampleSingleCell {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This class has 6 parents which is greater than 5 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)